specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NspAssociationList.json

```diff
@@ -37,7 +38,8 @@
               "profile": {
                 "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityPerimeters/nsp1/profiles/{profileName}"
               },
-              "accessMode": "DryRunMode"
+              "accessMode": "Audit",
+              "hasProvisioningIssues": "no"
```

Comment:
We can include another resource in the list response  to show a different value for this.
"hasProvisioningIssues": "yes"

Reply:
Added one object with "hasProvisioningIssues": "yes"